packages/test.sh

```diff
@@ -36,12 +36,23 @@ cd "$(dirname "$0")"
 TEST_PYTHON=${TEST_PYTHON:-"python3"}
 TEST_ENV_DIR=${TEST_ENV_DIR:-$(mktemp -d -t gxpkgtestenvXXXXXX)}
 
-"$TEST_PYTHON" -m venv "$TEST_ENV_DIR"
+if command -v uv >/dev/null; then
+    uv venv "$TEST_ENV_DIR" --python "$TEST_PYTHON"
+else
+    "$TEST_PYTHON" -m venv "$TEST_ENV_DIR"
+fi
+
 # shellcheck disable=SC1091
 . "${TEST_ENV_DIR}/bin/activate"
-pip install --upgrade pip setuptools wheel
+if ! command -v uv >/dev/null; then
+    pip install --upgrade pip setuptools wheel
+fi
 if [ $FOR_PULSAR -eq 0 ]; then
-    pip install -r../lib/galaxy/dependencies/pinned-typecheck-requirements.txt
+    if command -v uv >/dev/null; then
+        uv pip install -r ../lib/galaxy/dependencies/pinned-typecheck-requirements.txt
+    else
+        pip install -r ../lib/galaxy/dependencies/pinned-typecheck-requirements.txt
+    fi
```

Comment:
Can we define a `PIP` variable to either `uv pip` or `pip` and reuse that throughout the script to avoid duplicating every command?